docs/basics/userguide/seeding.md

```diff
@@ -378,3 +412,5 @@ run 3 :
 ```none
 [693246422, 3606543353, 433394544, 2194426398, 3928404622]
 ```
+
+<span>&#9888;</span> **warning :** If you like to fit an [ExperimentManager](rlberry.manager.ExperimentManager) with a [torch](https://pytorch.org/tutorials/beginner/deep_learning_60min_blitz.html) agent, you don't need to use the `set_external_seed`, rlberry do it for you.<span>&#9888;</span>
```

Comment:
If you like to fit an ... -> If you fit an ...

Reply:
rlberry do it for you -> rlberry does it for you

Reply:
done !